contracts/epochs/FlowEpoch.cdc

```diff
@@ -460,21 +468,31 @@ pub contract FlowEpoch {
 
     /// Calculates a new token payout for the current epoch
     /// and sets the new payout for the next epoch
-    access(account) fun calculateAndSetRewards(_ newPayout: UFix64?) {
+    access(account) fun calculateAndSetRewards() {
+
+        let stakingAdmin = self.borrowStakingAdmin()
 
-        let rewardsBreakdown = self.borrowStakingAdmin().calculateRewards()
+        // Calculate rewards for the current epoch that is about to end
+        // and save that reward breakdown in the epoch metadata for the current epoch
+        let rewardsBreakdown = stakingAdmin.calculateRewards()
         let currentMetadata = self.getEpochMetadata(self.currentEpochCounter)!
         currentMetadata.setRewardAmounts(rewardsBreakdown)
         self.saveEpochMetadata(currentMetadata)
 
-        // Calculate the new epoch's payout
-        // disabled until we enable automated rewards calculations
-        // let newPayout = FlowToken.totalSupply * (FlowEpoch.configurableMetadata.FLOWsupplyIncreasePercentage / 52.0)
+        if FlowEpoch.automaticRewardsEnabled() {
+            // Calculate the total supply of FLOW after the current epoch's payout
+            // the calculation includes the tokens that haven't been minted for the current epoch yet
+            let currentPayout = FlowIDTableStaking.getEpochTokenPayout()
+            let feeAmount = FlowIDTableStaking.getFeesBalance()
+            let flowTotalSupplyAfterPayout = FlowToken.totalSupply + (currentPayout - feeAmount)
 
-        if let payout = newPayout {
-            self.borrowStakingAdmin().setEpochTokenPayout(payout)
+            // Calculate the payout for the next epoch
+            let proposedPayout = flowTotalSupplyAfterPayout * FlowEpoch.configurableMetadata.FLOWsupplyIncreasePercentage
```

Comment:
It seems like this PR is changing the meaning of the `FLOWsupplyIncreasePercentage` config value, but the [documentation](https://github.com/onflow/flow-core-contracts/blob/master/contracts/epochs/FlowEpoch.cdc#L215-L217) isn't changed. 

The [documentation](https://github.com/onflow/flow-core-contracts/blob/master/contracts/epochs/FlowEpoch.cdc#L215-L217) says it is an annualized percentage yield value. Here we're using it as a per-payout percentage yield value. We should choose one of these options and make sure the documentation is consistent.

Reply:
yeah, that is what it originally was, but we decided to just make it constant for weekly increases. I'll update the comments